OxfordBodleian/Juel-Jensen/BDLaethg32.xml

```diff
@@ -37,9 +37,152 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    ወመላእክት<hi rend="rubric" rendition="#partialRubric">፤</hi> በቅድመ፡ ኵሎሙ፡ ደቂቀ፡ አዳም፡ ወሔዋን<hi rend="rubric" rendition="#partialRubric">፤</hi> ለገብርከ፡ <space/></explicit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="4r" to="4v"/>
+                                <title type="complete" ref="LIT6706PPSalotaHabt#ExplicitDirective"/>
+                                <textLang mainLang="gez" otherLangs="am"/>
+                                <note>The following is the entire text.</note>
+                                <incipit xml:lang="gez">ድግም፡ በሜሮን<hi rend="rubric" rendition="#partialRubric">፤</hi>
+                                    <hi rend="rubric" rendition="#partialRubric">፵፱</hi> ጊዜ፡ በቀለመ፡ ወርቅ፡ ጽፈሕ፡ ከከርቤ፡
+                                    ጋር፡ ታጠን<hi rend="rubric" rendition="#partialRubric">፨</hi> ዓዲ፡ 
+                                    <add place="above">በ</add>ነጭ፡ ዶሮ፡ መረቅ፡ በቀይ፡ ቀለም፡ ጽፈሕ፡ ጨምረህ<hi rend="rubric" rendition="#partialRubric">፤</hi>
+                                    አስርተህ፡ ብላ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                    መ<pb n="4v"/>ንካውን፡ ወጭቱን፡ ላባውን፡ ጥፍሩን፡ የታጠብክበትን፡ የማይበላውን፡ ሁሉ፡ ከግብ፡ ቅበር<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                    ከአምባጨ፡ ተቀጽላ፡ ክታቡን፡ ያዝ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                    <hi rend="rubric" rendition="#partialRubric">፨</hi> <hi rend="rubric" rendition="#partialRubric">፨</hi></incipit>
+                            </msItem>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus from="4v" to="6r"/>
+                            <title type="complete" ref="LIT6707PPBuryingHearts"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">አዳታኤል፡ ስምከ፡ ዘያድለቀልቃ፡
+                                ለምድር፡ ሰዱቃኤል፡</hi> ስምከ፡ ዘይስጥቃ፡ ለባሕር<hi rend="rubric" rendition="#partialRubric">፤</hi>
+                                ወኤቴኤል፡ ስምከ፡ ዘያቀልጦ፡ ለሰማይ<hi rend="rubric" rendition="#partialRubric">፤</hi> በዝ፡
+                                አስማት፡ ድፍን፡ ልቦሙ፡ ለሰብአ፡ እገሌ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                ያሙጅጅ<hi rend="rubric" rendition="#partialRubric">፤</hi> ያሙጅብ<hi rend="rubric" rendition="#partialRubric">፤</hi>
+                                አል<pb n="5r"/>ጀባር፡ አልዣባር<hi rend="rubric" rendition="#partialRubric">፤</hi>
+                                ገልማላዊ<hi rend="rubric" rendition="#partialRubric">፤</hi> አልሐዲር<hi rend="rubric" rendition="#partialRubric">፤</hi>
+                            </incipit>
+                            
+                            <explicit xml:lang="gez">ወያረስኦሙ፡ ምክሮሙ፡ ለመላእክት<hi rend="rubric" rendition="#partialRubric">፤</hi>
```

Comment:
one is the directive. I was not sure how to encode it. We could create a new subtype "directive" for explicit?

Reply:
That sounds reasonable to me. It was a question out of simply not understanding why you had two explicits, so it there is a functional distinction that needs to be made (I mean, if the "directive" needs to be included next to the "explicit"), then perhaps it would be good to be able to markup both of them specifically :)